### PR TITLE
Create page for mozilla.

### PR DESCRIPTION
**Pull request by maciejczyzewski**
This page should not be empty. I created a simple site presenting
mozilla activity on Github. It's good supplement instead of a 404 error.

Features:
- **members**
- **projects**

Example

Review it. @potch @ozten @zaach @tofumatt

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+============================== Presenting Mozilla Github Homepage ===========================
+------------------- The new central hub for Mozilla Open sources projects. ------------------
+
+                                ~~ Mozilla + Open Source = ♥ ~~
+
+      More information can be found in https://github.com/mozilla/mozilla.github.io
+
+======================= Join the team! ===> https://github.com/mozilla ======================
+
+                  Okay, now let's take a look on how this website is codded!
```
Comment (peterbe) [inline index.html]: coded

Comment (maciejczyzewski) [inline index.html]: fixed

File: README.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
-mozilla.github.io
-=================
+# Mozilla - Github
 
-The Mozilla Project uses GitHub!
+This page presents the activity of __mozilla__ on __Github__.
```
Comment (tofumatt) [inline README.md]: Mozilla is not lowercase.

Comment (maciejczyzewski) [inline README.md]: Yep, but often https://github.com/mozilla/sweet.js/blob/gh-pages/_layouts/default.html#L117

**Pull request closed without merging**